src/main/resources/application.properties

```diff
@@ -89,3 +89,6 @@ demand.request.notification.template=demandCreatetRequestTemplate
 
 announcement.default.search.pageSize=15
 
+sb.search.service.host=http://192.168.1.5:6007
```

Comment:
this should  "http://search-service:9000"

Reply:
made this change